bin/wiby.js

```diff
@@ -31,6 +31,5 @@ yargs
   )
   .help()
   .strict()
```

Comment:
```suggestion
  .strict()
  .parse()
```

The `.argv` getter performs parsing - it is the thing that executes the actual commands - without either `.argv` or `.parse()` things don't work.

Reply:
thanks!